tunneld/api.go

```diff
@@ -32,6 +36,7 @@ func (api *API) Router() chi.Router {
 		httpmw.RateLimit(10, 10*time.Second),
 	)
 
+	r.Post("/tun", api.postTun)
 	r.Post("/api/v1/clients", api.postClients)
```

Comment:
Should we consider introducing a new v2 endpoint?

Reply:
Do you mean changing `/api/v1` to be `/api/v2`? The `/api/v1/clients` endpoint is the v2 version of the old `/tun` endpoint.